RISK-AUTHENTICATION.md

```diff
@@ -0,0 +1,19 @@
+# Guide for Using Risk-Based Authentication With the Widget
```

Comment:
would you please make a new `docs` folder and move it to there, also use camel case for the file name, thanks.

Reply:
Changed.